src/Components/Navbar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Logo } from '../Components/Logo'
+import { Menu } from '../Components/Menu';
+
+function Navbar() {
```

Comment:
goooooood abstraction, dont be affraid of using components inside components inside components, they improve readability